explorer/src/serializers/chainlinkNodeSerializer.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  Serializer as JSONAPISerializer,
+  SerializerOptions,
+} from 'jsonapi-serializer'
+import { ChainlinkNode } from '../entity/ChainlinkNode'
+
+export const BASE_ATTRIBUTES: Array<string> = ['id', 'name', 'url', 'createdAt']
+
+const chainlinkNodeSerializer = (chainlinkNode: ChainlinkNode) => {
```

Comment:
I feel like this should be attached as a `toJSON` method on the models we want to serialize. `JSON.stringify` automatically calls this method when used on an object that has it defined.

Reply:
We should have serialization tests too.